go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/trussworks/terraform-aws-s3-private-bucket
 go 1.14
 
 require (
-	github.com/aws/aws-sdk-go v1.33.11
+	github.com/aws/aws-sdk-go v1.33.7
```

Comment:
Interesting. Why'd this need to be rolled back?

Reply:
Because I want dependabot to trigger. Easier to just roll back to an old version than wait for who knows how long for dependabot to find the next update.